src/main/java/com/yurjinia/common/security/jwt/filters/JwtAuthenticationFilter.java

```diff
@@ -33,6 +34,7 @@ public class JwtAuthenticationFilter extends OncePerRequestFilter {
 
     private final JwtService jwtService;
     private final UserDetailsService userDetailsService;
+    private final JwtBlacklistService jwtBlacklistService;
```

Comment:
the same here, must use methods from `JwtService` that call methods from `JwtBlacklistService`.